### ISSUE TITLE
Images causing text overlap on iPhone 6; fine on iPad

### ISSUE THREAD

**Issue by jamesabruce** [alleyinteractive/apple-news#100]
Our images are 640px wide; when previewed on iPad, they are correctly aligned to the right, following the plugin's default "anchor-layout-right" positioning. However, when previewed on an iPhone, they appear to be causing text overlap. 

In the screenshot attached, the overlapping text is supposed to be immediately after the heading, then an image, then the next paragraph. However, we end up with heading , image, text + text overlaid. 

![img_3534](https://cloud.githubusercontent.com/assets/2298025/13926935/7420b652-ef86-11e5-85ec-b20ea5f9b90c.PNG)

This is the content that's causing it (though this is happening on all posts, not just this one)

```
<h3><a href="http://www.flickr.com/">Flickr</a></h3>
While it's generally thought of as an <a href="http://www.makeuseof.com/tag/online-photo-albums-host-free/">image-sharing site</a>, Flickr is also great for photo storage and organization, mostly because of the fact that you receive an entire terabyte of storage for free when you sign up for a free account.

<img class="aligncenter" src="http://www.makeuseof.com/wp-content/uploads/2016/03/flickr-photostream.png" alt="flickr-photostream" width="640" height="437" />

You also get editing tools powered by Aviary; they're not the best, but you can do all of the basics that you're likely to need, like adjusting brightness and contrast, and getting rid of red-eye.

```

**Comment by jamesabruce** [comment]
I've also just noticed it's stripping the link from the header, but this might be intentional to comply with style standards.

**Comment by bcampeau** [comment]
The removal of the link is expected.

We're looking into some layout issues this week and we'll see if this is related.

**Comment by bcampeau** [comment]
@jamesabruce I've looked at this from many different angles and cannot replicate this bug. Which exact iPhone model did you see this on? Is it still happening with version 1.0.7 of the plugin? Can you send me a link to this post on your site?

Thanks.

**Comment by bcampeau** [comment]
@jamesabruce also, if possible, could you go to the post editor and paste the content of the "Text" tab either here in a code snippet or in a text file?

Thanks!

**Comment by jamesabruce** [comment]
@bcampeau Seems to fixed in 1.0.7, thank you!
